apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -8,7 +8,10 @@ import PropTypes from 'prop-types';
 import Radium from 'radium';
 import {connect} from 'react-redux';
 import i18n from '@cdo/locale';
-import AgeDialog, {signedOutOver13} from '../templates/AgeDialog';
+import AgeDialog, {
+  signedOutUnder13 as signedOutAgeCheck,
```

Comment:
`signedOutAgeCheck` seems less clear than `signedOutUnder13`.  Do we really need to alias the function here?

Reply:
I ended up adding the alias because otherwise there is a naming collision on line 99 `const signedOutUnder13 = signedOutAgeCheck();` I like having the const named `signedOutUnder13` there because I think it captures the logic really well since we use it in booleans with `signedInUnder13`. Aliasing the function name here felt the 'least bad' way out of that naming collision. That being said, if you have other suggestions to circumvent this, please let me know!

Reply:
Given that it's assigned to a variable and then only used once on the next line, I think I'd just avoid the temporary assignment to a variable and have
```
return signedInUnder13 || signedOutOver13();
```

Agreed that it's nice to read two variables in the `||`, but I think the reduction in clarity in the name `signedOutAgeCheck` is a bigger downside.

If you wanted to keep the two things symmetrical, I guess you could create a `signedInUnder13()` function.

Reply:
If we're cool with this: `return signedInUnder13 || signedOutOver13();` I think that is easy to read and circumvent this issue perfectly. :+1: